foundation/foundation/README.md

```diff
@@ -28,7 +28,7 @@ components are mapped in the base theme. It is your responsibility to make all c
 bindings in your custom theme.
 
 ```xml
-<style name="Your.AppTheme" parent="Base.Theme.Vitamin">
+<style name="Your.AppTheme" parent="Base.Theme.Vitamin.Foundation.x">
```

Comment:
I think you can specify `Light` and `Dark` here instead of redirect to `vitamin` module here.

Foundation artifact cover really specific cases from my opinion.